### PR TITLE
Feature/MaganProps

### PR DESCRIPTION
**Pull request by m4rtint**
- Ball
- Beach Table Umbrella
- Candy Cane
- Gingerbread Man
- Monkey Bars
- Seesaw
- Shovel
- Snowman
- Soccer Goal

### CODE REVIEW

**Comment by m4rtint** [comment]
[Import props Magan created (0.5)](https://trello.com/c/6mfz1Nca/756-import-props-magan-created-05)

**Comment by m4rtint** [comment]
[Import props Magan created (0.5)](https://trello.com/c/6mfz1Nca/756-import-props-magan-created-05)

**Comment by m4rtint** [comment]
[Create more Particle FX (1.0)](https://trello.com/c/HAii6Rs3/755-create-more-particle-fx-10)

File: Assets/_PartyParkade/Script/Game/Destructibles/GenericDestructibleBehaviour.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 //     Copyright (c) PERIGON GAMES. 2020 All Rights Reserved
 // </copyright>
 //-----------------------------------------------------------------------
+
```
Comment (Gursaanj) [inline Assets/_PartyParkade/Script/Game/Destructibles/GenericDestructibleBehaviour.cs:6]: Doesnt Look like the name changes were comitted

**Comment by m4rtint** [comment]
[More destructible objects](https://trello.com/c/P6o6ppBr/778-more-destructible-objects)

**Pull request merged**